### DISCUSSION TITLE
v2.2.0.beta.0

### DISCUSSION THREAD

**Discussion by neverhood311** [neverhood311/Stop-motion-OBJ#168] [General]
### Major features

- Keyframed sequence progression
- Manual sequence creation (for stop motion animation)
- Smooth/Flat shading for Streaming Sequences
- Auto-export modified meshes
- Multiple sequence import
- .X3D mesh sequences

### Minor features

- Display mesh sequence source path in the Advanced panel
- Display sequence length in the Advanced panel
- Display the number of currently-cached meshes
- Merge duplicate materials

<hr /><em>This discussion was created from the release <a href='https://github.com/neverhood311/Stop-motion-OBJ/releases/tag/v2.2.0.beta.0'>v2.2.0.beta.0</a>.</em>

**Comment by ge53** [comment]
this works beautifully--thanks for sharing...is there any way to edit the mesh exported as 'mesh sequence' other than your suggestion of parenting the object to an empty and transforming --i know i can use a different add-on to change frames to individual shape keys but that is crazy for an animated sequence..
any tips/suggestions welcome!

**Comment by A-intheCode** [comment]
Hey fantastic release it works perfectly with Blender 3.6.5 however there is a problem when importing obj sequences with Color Data on Point Domain. The color will not be applied or even imported with the sequence. Importing the data with the native blender obj importer works well. Can you please try to provide color import like with the native blender importer?i have a temporary solution via geometry nodes. I really like your development and contribution. This addon is fantastic

**Comment by PolygardenStudio** [comment]
I was trying to find a way to contact you directly about this project, but I couldn't find a way to send you a private message. I am a grad student working on volumetric capture and this tool so far might be a good solution for me to transfer the export of the animation to get it into a game engine. This new version is working well for Blender 3, but are you planning on a version for Blender 4?

**Reply by neverhood311** [reply]
The upgrade to Blender 4 should have been a simple one, but they've made a change to their OBJ importer that makes it basically impossible to do Streaming Sequences. I've thought about writing my own compatible OBJ importer but that's an undertaking that I haven't had the time for lately.
